app/assets/stylesheets/read/bookmark.less

```diff
@@ -0,0 +1,8 @@
+.leaflet-bookmark {
+  margin-right: 30px !important;
```

Comment:
I think you could get away from `!important` by selecting something like:

```
.leaflet-control.leaflet-bookmark
```

Checkout: https://github.com/sul-cidr/graves/blob/master/app/assets/stylesheets/read/reset-button.less#L1